### PR TITLE
update UI text strings for code cleanup setting

### PR DESCRIPTION
**Pull request by JieCarolHu**
<details><summary>Ask Mode template not completed</summary>

### Customer scenario

What does the customer do to get into this situation, and why do we think this
is common enough to address for this release.  (Granted, sometimes this will be
obvious "Open project, VS crashes" but in general, I need to understand how
common a scenario is)

### Bugs this fixes

(either VSO or GitHub links)

### Workarounds, if any

Also, why we think they are insufficient for RC vs. RC2, RC3, or RTW

### Risk

This is generally a measure our how central the affected code is to adjacent
scenarios and thus how likely your fix is to destabilize a broader area of code

### Performance impact

(with a brief justification for that assessment (e.g. "Low perf impact because no extra allocations/no complexity changes" vs. "Low")

### Root cause analysis

How did we miss it?  What tests are we adding to guard against it in the future?

### How was the bug found?

(E.g. customer reported it vs. ad hoc testing)

### Test documentation updated?

If this is a new non-compiler feature or a significant improvement to an existing feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.

</details>

### CODE REVIEW

**Comment by JieCarolHu** [comment]
image

File: src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.cs.xlf [hunk 1]
```diff
@@ -2,59 +2,54 @@
...
+        <source>Apply all C# formatting rules (indentation, wrapping, spacing)</source>
+        <target state="new">Apply all C# formatting rules (indentation, wrapping, spacing)</target>
         <note />
       </trans-unit>
-      <trans-unit id="Fix_expression_bodied_members">
-        <source>Fix expression-bodied members</source>
-        <target state="new">Fix expression-bodied members</target>
+      <trans-unit id="Apply_expression_bodied_members_preferences">
+        <source>Apply expression-bodied member preferences</source>
+        <target state="new">Apply expression-bodied member preferences</target>
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.cs.xlf]: perhaps: Apply expression/block body preferences

**Comment by JieCarolHu** [comment]
retest windows_debug_vs-integration_prtest

**Comment by heejaechang** [comment]
looks good to me. but I probably not a good person for naming. @jinujoseph ?

**Comment by jinujoseph** [comment]
This needs to be reviewed by @kuhlenh

**Comment by kuhlenh** [comment]
LGTM (based on screenshot and the expression/block change)

**Comment by JieCarolHu** [comment]
@jinujoseph Could you approve the merge? Thanks

**Pull request merged**